File: frontend/src/models/dto/HallRow.js
export class HallRow {
    /**
     * @typedef {{
     *     id: Number,
     *     hallId: Number,
     *     number: Number,
     *     count: String,
     *     hint: Number,
     *     categoryId: Number,
     * }} JavaHallRow
     */

     /**
     * @param model {JavaHallRow}
     */
    constructor(model) {
        this._id = model.id;
        this._hallId = model.hallId;
        this._number = model.number;
        this._count = model.count;
        this._hint = model.hint;
        this._categoryId = model.categoryId;
    }

    get id() {
        return this._id;
    }

    set id(value) {
        this._id = value;
    }

    get hallId() {
        return this._hallId;
    }

    set hallId(value) {
        this._hallId = value;
    }

    get number() {
        return this._number;
    }

    set number(value) {
        this._number = value;
    }

    get count() {
        return this._count;
    }

    set count(value) {
        this._count = value;
    }

    get hint() {
        return this._hint;
    }

    set hint(value) {
        this._hint = value;
    }

    get categoryId() {
        return this._categoryId;
    }

    set categoryId(value) {
        this._categoryId = value;
    }
}

File: src/main/java/com/practice/theater/models/xml/HallRow.java
package com.practice.theater.models.xml;

import com.practice.theater.db.annotations.Column;
import com.practice.theater.db.annotations.Table;

@Table(name = "HallRows")
public class HallRow {
    @Column(name = "id", isPrimaryKey = true)
    private long id;
    @Column(name = "hallId")
    private long hallId;
    @Column(name = "number")
    private int number;
    @Column(name = "count")
    private int count;
    @Column(name = "hint")
    private String hint;
    @Column(name = "categoryId")
    private long categoryId;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getHallId() {
        return hallId;
    }

    public void setHallId(long hallId) {
        this.hallId = hallId;
    }

    public int getNumber() {
        return number;
    }

    public void setNumber(int number) {
        this.number = number;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public String getHint() {
        return hint;
    }

    public void setHint(String hint) {
        this.hint = hint;
    }

    public long getCategoryId() {
        return categoryId;
    }

    public void setCategoryId(long categoryId) {
        this.categoryId = categoryId;
    }
}
